installers/win/hobovr_installer.nsi

```diff
@@ -245,41 +72,32 @@ UninstPage instfiles
 Function .onInit
   StrCpy $isexpress "true"
   StrCpy $INSTDIR "$PROGRAMFILES\Hobo VR"
-  StrCpy $minPythonMinorVer 7
 FunctionEnd
 
-
 Section
-
-CreateDirectory $INSTDIR
-AccessControl::GrantOnFile  "$INSTDIR" "(S-1-5-32-545)" "FullAccess"
-
-Call EnsurePython
-
-# Download and Unzip HoboVR Master branch to install directory
-inetc::get "https://github.com/okawo80085/hobo_vr/archive/refs/heads/master.zip" "$TEMP\hobovr_master.zip" $HobovrZipDownloaded
-Pop $R0
-StrCpy $HobovrZipDownloaded $R0
-StrCmp $HobovrZipDownloaded "OK" 0 hobo_download_failed
-nsisunz::Unzip "$TEMP\hobovr_master.zip" "$INSTDIR"
 	
 # Tell Steam where the VR DLLs are
-Call EnsureSteam
-
-IfFileExists $VRPathRegEXE PathGood
-	MessageBox MB_OK|MB_ICONEXCLAMATION "Steam VR path doesn't exist. has steam and steamvr been installed?"
+IfFileExists "C:\Program Files (x86)\Steam\steamapps\common\SteamVR\bin\win32\vrpathreg.exe" PathGood
+	MessageBox MB_OK|MB_ICONEXCLAMATION "Steam vr path register exe doesn't exist. has steam and steamvr been installed?"
 	Quit
 PathGood:
-	ExecDos::exec /DETAILED '"$VRPathRegEXE" adddriver "$INSTDIR\hobo_vr-master\hobovr"' $EXIT_CODE install_log.txt
-	ExecDos::exec /DETAILED '"$VRPathRegEXE" show' $EXIT_CODE install_log.txt
+    # Create Directory
+    CreateDirectory $INSTDIR
+    AccessControl::GrantOnFile  "$INSTDIR" "(S-1-5-32-545)" "FullAccess"
+    # Unpack the files
+    SetOutPath $INSTDIR
+    File /a /r "..\..\hobovr\*.*"
+
+	ExecDos::exec /DETAILED '"C:\Program Files (x86)\Steam\steamapps\common\SteamVR\bin\win32\vrpathreg.exe" adddriver "$INSTDIR\hobovr"' $EXIT_CODE install_log.txt
+	ExecDos::exec /DETAILED '"C:\Program Files (x86)\Steam\steamapps\common\SteamVR\bin\win32\vrpathreg.exe" show' $EXIT_CODE install_log.txt
 
 # Install hobovr virtualreality python bindings
-ExecDos::exec /DETAILED '$pythonExe -m pip install -e "$INSTDIR\hobo_vr-master\bindings\python"' $EXIT_CODE install_log.txt
+# ExecDos::exec /DETAILED '$pythonExe -m pip install -e "$INSTDIR\hobo_vr-master\bindings\python"' $EXIT_CODE install_log.txt
 goto end
 	
 # Handle errors
-hobo_download_failed:
-	MessageBox MB_OK|MB_ICONEXCLAMATION "Unable to download hobo_vr: $HobovrZipDownloaded."
+hobo_failed:
```

Comment:
Does _this_ work though? I don't think that `goto end` will ever be skipped. If there are crashing errors, the program would just exit instead.